candi/cloud-providers/aws/openapi/cluster_configuration.yaml

```diff
@@ -88,7 +88,7 @@ apiVersions:
             x-doc-example: |
               ```yaml
               project: cms-production
-              severity: critical
+              severity_level: 3
```

Comment:
Oh, let's leave critical there. It is an example of possible additional tags on cloud instances.
Examples in the `candi/` and `ee/candi/` folders should stay the same.